extensions/sql-database-projects/src/models/project.ts

```diff
@@ -117,6 +117,10 @@ export class Project implements ISqlProject {
 		return this._sqlProjStyle;
 	}
 
+	public get sqlProjStyleName(): string {
```

Comment:
I don't love this, but given the confusion around typing file references for the migration extension earlier, I didn't want to add an mssql-vscode or mssql dependency now.  I'd like for this to be able to be replaced by the enum with the reintroduction of vscode support.